test/tutorials/django-ci.sh

```diff
@@ -12,9 +12,28 @@ cd examples/ecs-fargate/django/app
 echo "\nInstall pip packages"
 pip install -r requirements.txt
 
-echo "\nRun Django migrations"
+# install ecs_simplified and codebuild modules
+psql "postgres://$IASQL_USERNAME:$IASQL_PASSWORD@localhost:5432/iasql" -c "
+  SELECT * FROM iasql_install(
+      'aws_ecs_simplified', 'aws_codebuild'
+  );
+"
+
+echo "\nRun Django migrations - no real action on the database"
```

Comment:
Is this a requirement of Django itself? (Eg, if we comment this out follow-up stuff doesn't work?)

Reply:
No. We can omit it and nothing will fall apart.